src/integrations/Klaviyo/browser.js

```diff
@@ -141,9 +141,10 @@ class Klaviyo {
 
   track(rudderElement) {
     const { message } = rudderElement;
-    const { event } = message;
+    let { event } = message;
     const properties = message?.properties || {};
 
+    event = event ? event.trim().toLowerCase() : event;
```

Comment:
To use `const` variables as much as possible
```suggestion
    event = event ? event.trim().toLowerCase() : event;
```
const event = message.event ? message.event.trim().toLowerCase() : message.event;

Reply:
Udpated